controlplane/kubeadm/internal/workload_cluster_conditions_test.go

```diff
@@ -1449,9 +1476,10 @@ func withMachineReadyV1beta2Condition(status metav1.ConditionStatus) fakeMachine
 			machine.Status.V1Beta2 = &clusterv1.MachineV1Beta2Status{}
 		}
 		machine.Status.V1Beta2.Conditions = append(machine.Status.V1Beta2.Conditions, metav1.Condition{
-			Type:   clusterv1.MachinesReadyV1Beta2Condition,
-			Status: status,
-			Reason: "SomeReason",
+			Type:    clusterv1.MachinesReadyV1Beta2Condition,
```

Comment:
```suggestion
			Type:    clusterv1.MachineReadyV1Beta2Condition,
```
Let's use this one please. Ready is a lot less confusing than using MachinesReady here (as this is just a single Machine and not a MD/MS/KCP)